integration/test_integration_basics.py

```diff
@@ -29,6 +29,31 @@ def test_h1(self):
     self.assertEqual(counters["upstream_rq_total"], 25)
     self.assertEqual(len(counters), 9)
 
+  def mini_stress_test_h1(self, args):
+    # run a test with more rps then we can handle, and a very small client-side queue.
+    # we should observe both lots of successfull requests as well as time spend in blocking mode.,
+    parsed_json = self.runNighthawkClient(args)
+    counters = self.getNighthawkCounterMapFromJson(parsed_json)
+    self.assertGreater(counters["benchmark.http_2xx"], 1000)
```

Comment:
This (and below) are coupled with the call site, maybe try and have one source of truth for the number of reqs.

Reply:
Done